addonscript/test/test_task.py

```diff
@@ -104,3 +117,57 @@ def test_build_filelist_missing_file(context, task_dfn):
     context.task = task_dfn
     with pytest.raises(TaskVerificationError):
         task.build_filelist(context)
+
+
+@pytest.mark.parametrize('api_root, scope, raises', (
+    ('http://localhost:5000', 'project:releng:addons.mozilla.org:server:dev', False),
+    ('http://some-amo.url', 'project:releng:addons.mozilla.org:server:dev', False),
+    ('https://addons.allizom.org', 'project:releng:addons.mozilla.org:server:staging', False),
+    ('https://addons.allizom.org/', 'project:releng:addons.mozilla.org:server:staging', False),
+    ('https://addons.mozilla.org', 'project:releng:addons.mozilla.org:server:production', False),
+    ('https://addons.mozilla.org/', 'project:releng:addons.mozilla.org:server:production', False),
+))
```

Comment:
I don't see any raises: True here

Reply:
Oh that's right. I cleaned that up. The failures are handled in the next function, anyway.